Sources/SnowballSwiftKit/Views/Chip.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  Chip.swift
+//
+//
+//  Created by Vivian Phung on 6/30/23.
+//
+
+import SwiftUI
+import SnowballAssetKit
+
+public enum ChipSize {
+    case small
+    case medium
+}
+
+public struct Chip: View {
+    @Binding var isActive: Bool
+    var isDisabled: Bool
+    var title: String
+    var leftIcon: String?
+    var leftSystemIcon: String?
+    var rightIcon: String?
+    var rightSystemIcon: String?
+    var size: ChipSize
+
+    public init(isActive: Binding<Bool> = .constant(true),
+                isDisabled: Bool = false,
+                title: String,
+                leftIcon: String? = nil,
+                leftSystemIcon: String? = nil,
+                rightIcon: String? = nil,
+                rightSystemIcon: String? = nil,
+                size: ChipSize = .small) {
+        self._isActive = isActive
+        self.isDisabled = isDisabled
+        self.title = title
+        self.leftIcon = leftIcon
+        self.leftSystemIcon = leftSystemIcon
+        self.rightIcon = rightIcon
+        self.rightSystemIcon = rightSystemIcon
+        self.size = size
+    }
+
+    public var body: some View {
+        Button {
+            isActive.toggle()
+        } label: {
+            HStack(spacing: 4) {
+                if let leftIcon = leftIcon {
+                    Image(leftIcon)
+                }
+                if let leftSystemIcon = leftSystemIcon {
+                    Image(systemName: leftSystemIcon)
+                }
```

Comment:
it appears you can have two icons.

we probably don't want to do it this way and instead have left / right accessory views.